api/src/services/Fedora.ts

```diff
@@ -310,6 +310,42 @@ export class Fedora {
         }
     }
 
+    /**
+     * This method removes an isMemberOf relationship from a pid/parent pid pair.
+     *
+     * @param pid       PID to update
+     * @param parentPid Parent PID to update
+     */
+    async deleteParentRelationship(pid: string, parentPid: string): Promise<void> {
+        const predicate = "info:fedora/fedora-system:def/relations-external#isMemberOf";
+        const targetPath = "/" + pid;
+        const insertClause = "";
```

Comment:
Are these empty strings for a reason? Can you just add the empty string into the parameters?